frontend/src/utils/withItem.js

```diff
@@ -80,9 +82,12 @@ export const withContainer = createWithItem(Container, api.containers)
 export const withSample = createWithItem(Sample, api.samples)
 export const withIndividual = createWithItem(Individual, api.individuals)
 export const withUser = createWithItem(User, api.users)
+export const withProcess = createWithItem(Process, api.processes)
+export const withProtocol = createWithItem(Protocol, api.protocols)
```

Comment:
Not sure this will ever be useful since we load all protocols each time we reload the site... If you see a use for it, you could add it to the export below too.